sandpack-react/src/presets/Sandpack.tsx

```diff
@@ -103,12 +26,12 @@ export const Sandpack: React.FC<SandpackProps> = (props) => {
     showReadOnly: props.options?.showReadOnly,
   };
 
-  const providerOptions: SandpackProviderProps = {
+  const providerOptions: SandpackOptions = {
     openPaths: props.options?.openPaths,
     activePath: props.options?.activePath,
     recompileMode: props.options?.recompileMode,
     recompileDelay: props.options?.recompileDelay,
-    autorun: props.options?.autorun ?? true,
+    autorun: props.options?.autorun,
```

Comment:
haven't searched for this, but doesn't the autorun need to be `true` by default?

Reply:
Yes, but I leave the SandpackContext to set the default values